stdlib/3/io.pyi

```diff
@@ -164,13 +164,15 @@ class TextIOWrapper(TextIO):
     #              -> None: ...
     def __init__(
         self,
-        buffer: IO[bytes],
+        buffer: BinaryIO,
```

Comment:
I believe CI fails due to this change. Honestly, I would just leave this instance as `IO[bytes]` for now. In the future it might make sense to make `TextIOWrapper` generic over the buffer, but I haven't looked at it in depth.